requests-for-comments/2024-06-21-warehouse-launch-plan.md

```diff
@@ -0,0 +1,49 @@
+## [Marketing] Data warehouse launch plan
+
+The data warehouse has been in an opt-in beta for a while, and has now been opened up to a public open beta. It'll be at least a few weeks before it launches (we need to give current users two weeks notice of pricing changes) BUT that means it's the perfect time to plan. 
+
+### Ahead of launch
+
+- [x] Publish a [Stripe data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8662) - Marketing
+- [x] Publish a [Hubspot data warehouse tutorial](https://github.com/PostHog/posthog.com/issues/8664) - Marketing
+- [x] Publish a [Zendesk data warehouse tutorial](https://github.com/PostHog/posthog.com/pull/8858) - Marketing
+- [ ] Create dashboard templates for 2x common usecases - @ivanagas
+- [x] Update docs as needed - Marketing
+- [ ] Add Data Warehouse to in-app 'Explore the docs' menu - https://github.com/PostHog/posthog/pull/22715
+- [x] Key art - @lottiecoxon (we have some already, open to new though)
+
+
+### Launch day activity
+
+- [ ] Publish a launch blog piece about why data warehouse is important - @joethreepwood 
+- [ ] Update pricing page with data warehouse pricing - Growth
+- [ ] Update in-app onboarding with data warehouse option - Growth
+- [ ] Share blogpost to HackerNews - Marketing
+- [ ] https://github.com/PostHog/meta/issues/213 - @joethreepwood
+- [ ] Social push, changelog, Pylon announcement - @joethreepwood
+
+### Follow-on activity
+
+- [ ] Case study: https://github.com/PostHog/posthog.com/issues/8684 - @joethreepwood
+- [ ] Update onboarding email flow with data warehouse options - @joethreepwood
+- [ ] Invite Theo/Danny to try the warehouse and give what they need for a video on it - @ivanagas
```

Comment:
@ivanagas We're currently targeting a July 8th launch. Was there any chance of this?

Reply:
I didn't follow up on this because I thought we wanted the launch to be quiet, this is going to show it to a lot of people, so I'm hesitant.

Reply:
Yep, all good - lets remove then.

Reply:
```suggestion
```